package.json

```diff
@@ -110,7 +110,7 @@
     ]
   },
   "description": "Smart contracts and dapp implementing EEA spec onchain permissioning",
-  "version": "2.1.0",
+  "version": "2.1.2",
```

Comment:
Just out of curiosity: why increment by 2 and not by one?

Reply:
because I manually did the release of 2.1.1 and did not commit this change in the meantime